proposals/lip-0048.md

```diff
@@ -25,15 +25,65 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
+
+The option to transfer the consumed fees to a dedicated pool address `ADDRESS_FEE_POOL` could be beneficial for sidechains, as fees collected this way can be used as an incentive for validators and other protocol participants for various purposes: Funding a community pool for on-chain governance spending, sharing funds proportionally among validators per round, incentivizing ecosystem developers, supporting chain maintenance and improvement, rewarding users' staking and voting activities, etc.
+
+In the Lisk mainchain, the consumed fees are always burned. This reduces the total supply, creating a deflationary pressure that works against inflation.
+
+### Cross-chain Message Fees
+
+Cross-chain messages (CCMs) facilitate interoperability by enabling cross-chain transfer of data. Each CCM contains a [*message fee*][lip-0049#ccmschema], intended to cover the cost of transaction processing on the receiving chain (see [LIP 0049][lip-0049#fee]). 
+
+Since managing message fees requires checking and updating the escrow balances for the message fee token, this task is assigned to the Token module whose [`payMessageFee`][lip-0051#paymessagefee] method charges message fees in the sending chain. In the receiving chain, the Token module assigns the message fee to the relayer account and updates escrows if necessary, and then the Fee module handles the transaction fee processing as described above. Note that the CCM sender should include any additional fees necessary for the CCM execution, e.g., to pay for the user account initialization on the receiving chain.
+
+#### Minimum Fee per CCM
+
+For protocol reasons (see, e.g., [`bounce`][lip-0045#bounce] and [`terminateChain`][lip-0045#terminatechain] methods in LIP 0045), the Interoperability module allows the message fee to be zero. Consequently, the choice of setting a minimum message fee is delegated to each module that provides its own cross-chain commands. Nevertheless, there is a minimum fee in case the user wants the failed CCM to be returned (see [`bounce`][lip-0045#bounce] method in LIP 0045). Thus, even though this fee is not mandatory, it is advisable to treat it as a suggested minimum fee. 
+
+#### Message Fee Processing
+
+Processing of message fees is similar to that of transaction fees. It should be noted that the consumed part of the message fee contains only the so-called additional fees, as there is no mandatory minimum fee. The consumed part can either be burned or transferred to a fee pool, depending on the chain configuration. 
+
+The available part of the message fee is left to the relayer to cover the cost of CCU execution and to incentivize its participation in CCM relaying. Also, since a CCU is a regular transaction, note that the relayer pays transaction fees for posting the CCU in the receiving chain. 
```

Comment:
```suggestion
The available part of the message fee is left to the relayer to cover the cost of CCU execution (recall that a CCU is a regular transaction, therefore the relayer has to pay transaction fees for posting the CCU in the receiving chain) and to incentivize its participation in CCM relaying. 
```

Reply:
Suggest this change because the last sentence seemed out of context compared to the first one.